lib/controller.js

```diff
@@ -967,3 +967,4 @@ module.exports = controller;
 // Shared exports
 module.exports.listTessels = controller.list;
 module.exports.getTessel = controller.get;
+module.exports.LOCAL_AUTH_KEY = Tessel.LOCAL_AUTH_KEY;
```

Comment:
- `Tessel.LOCAL_AUTH_KEY` isn't defined in this file, so should not be exporting it
- `module.exports.LOCAL_AUTH_KEY = Tessel.LOCAL_AUTH_KEY` breaks the accessor